docs/routing-rules.md

```diff
@@ -132,73 +138,77 @@ You can use the `contains` operator
+4. `Trino-UI-Token` or `__Secure-Trino-ID-Token` cookie.
 
 Kerberos and Certificate authentication are not currently supported. If the 
-request contains the `Authorization: Bearer` header, an attempt will be made to
-treat the token as a JWT and deserialize it. If this is successful, the 
+request contains the `Authorization: Bearer` header, an attempt is made to treat
+the token as a JWT and deserialize it. If this is successful, the
 value of the claim named in `requestAnalyzerConfig.tokenUserField` is used as
 the username. By default, this is the `email` claim. If the token is not a valid
 JWT, and `requestAnalyzerConfig.oauthTokenInfoUrl` is configured, then the token
-will be exchanged with the Info URL. Responses are cached for 10 minutes to
-avoid triggering rate limits. 
+is exchanged with the Info URL. Responses are cached for 10 minutes to avoid
+triggering rate limits.
 
 You may call `trinoRequestUser.getUser()` and `trinoRequestUser.getUserInfo()`
-in your routing rules. If a user was not successfully extracted,
-`trinoRequestUser.getUser()` will return an empty
-[Optional](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html).
-`trinoRequestUser.getUserInfo()` will return an
-[Optional\<UserInfo\>](https://www.javadoc.io/doc/com.nimbusds/oauth2-oidc-sdk/5.34/com/nimbusds/openid/connect/sdk/claims/UserInfo.html)
+in your routing rules. If user information was not successfully extracted,
+`trinoRequestUser.getUser()` returns an empty `Optional`.
+`trinoRequestUser.getUserInfo()` returns an `Optional<UserInfo>`, with an
+[OpenID Connect UserInfo](https://www.javadoc.io/doc/com.nimbusds/oauth2-oidc-sdk/5.34/com/nimbusds/openid/connect/sdk/claims/UserInfo.html)
 if a token is successfully exchanged with the `oauthTokenInfoUrl`, and an empty
 `Optional` otherwise.
 
 `trinoRequestUser.userExistsAndEquals("usernameToTest")` can be used to check a
-username against the extracted user. It will return `False` if a user has not
-been extracted.
+username against the extracted user. It returns `false` if a user has not been
+extracted.
 
 User extraction is only available if enabled by configuring
 `requestAnalyzerConfig.analyzeRequest = True`
 
 ### TrinoQueryProperties
 
-This class attempts to parse the body of a request as SQL. Note that only a
-syntactic analysis is performed! If a query œreferences a view, then that
-view will not be expanded, and tables referenced by the view will not be
-recognized. Note that Views and Materialized Views are treated as tables and
-added to the list of tables in all contexts, including  statements such as
-`CREATE VIEW`.
+The `TrinoQueryProperties` class attempts to parse the body of a request to
+determine the SQL statement and other information. Note that only a
+syntactic analysis is performed.
+
+If a query references a view, then that view  is expanded, and tables referenced
+by the view are recognized. Views and materialized views are treated as
```

Comment:
```suggestion
If a query references a view, then that view is not expanded, and tables referenced
by the view are not recognized. Views and materialized views are treated as
```
This is what is implied by syntactic-only analysis. The Trino Gateway does not reach out to the backend to get information on view and table definitions